bindings/wallet-jni/java/run_test.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import subprocess
+import sys
+
+scriptdirectory = Path(__file__).parent
+rootdirectory = scriptdirectory.parent.parent.parent
+dynlibdirectory = rootdirectory / "target" / "debug"
+
+junit = "junit-4.13.jar"
+hamcrest = "hamcrest-core-1.3.jar"
+
+classpath = f".:{junit}:{hamcrest}"
+javafilesdir = Path("com/iohk/jormungandrwallet")
+
+
+def compile():
+    packageFiles = map(lambda path: str(path), (scriptdirectory /
+                                                Path("com/iohk/jormungandrwallet")).glob("*.java"))
+    out = subprocess.run([
+        "javac", "-cp", classpath, "WalletTest.java"] + list(packageFiles), cwd=scriptdirectory)
```

Comment:
In `python3` you can also `["javac", "-cp", classpath, "WalletTest.java", *packageFiles]`, I think.

Reply:
yes, it works. I like it, I didn't know that